assets/source/setup-guide/app/views/ConnectionApp.js

```diff
@@ -54,14 +55,18 @@ const SettingsApp = () => {
 			{ appSettings ? (
 				<div className="woocommerce-setup-guide__container">
 					<SetupAccount
-						view="settings"
+						view={ SETTINGS_VIEW }
 						setIsConnected={ setIsConnected }
 						isConnected={ isConnected }
 						isBusinessConnected={ isBusinessConnected }
 					/>
 
-					{ isGroup1Visible && <ClaimWebsite view="settings" /> }
-					{ isGroup2Visible && <SetupTracking view="settings" /> }
+					{ isGroup1Visible && (
+						<ClaimWebsite view={ SETTINGS_VIEW } />
+					) }
+					{ isGroup2Visible && (
+						<SetupTracking view={ SETTINGS_VIEW } />
+					) }
```

Comment:
:nail_care: / :scroll: 
I think what we can still do here in regards to `context` and https://github.com/woocommerce/pinterest-for-woocommerce/issues/302 is to forward the `view` to  `SaveSettingsButton` and use it as the in `recordEvent`. So from the perspective of `SaveSettingsButton` it will set the context to whatever is given to it. Then #302 would need to simply fix the view prop being set to a distinct value.

Reply:
Fixed in https://github.com/woocommerce/pinterest-for-woocommerce/pull/295/commits/0afbde0966e4b9124d2ae8a573d788ea88213746